main.go

```diff
@@ -102,6 +104,43 @@ func main() {
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
 		LeaderElectionID:       "2793210b.openshift.io",
+        Cache: cache.Options{
+            DefaultNamespaces: map[string]cache.Config{
+                config.OperatorNamespace: {},
+            },
```

Comment:
Do we need to cache objects in the operator namespace? The per-object caches make sense to me. I'm reading through https://github.com/kubernetes-sigs/controller-runtime/blob/main/designs/cache_options.md#example-usages

Reply:
Not really, we could have a detailed list of every single object used, I need to add a couple more I think. In this specific case, I don't think it would make a significant difference.